frontend/services/api.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import Jsona from 'jsona';
+import { getSession, signOut } from 'next-auth/client';
+
+/**
+ * API service require to be authenticated.
+ * Then, by default all the request are sending the authorization header.
+ */
+
+const dataFormatter = new Jsona();
+
+const defaultConfig = {
+  baseURL: `${process.env.NEXT_PUBLIC_API_URL || process.env.STORYBOOK_API_URL}/api/v1/`,
```

Comment:
`NEXT_PUBLIC_API_URL` is already defined as `/backend/api/v1` in `.env.sample` so I don't think you should add `/api/v1/` here at the end.

Reply:
This will need to change once we merge #76 because of the new environment variables.